Tests/SwiftLintFrameworkTests/CommandTests.swift

```diff
@@ -223,6 +225,37 @@ class CommandTests: XCTestCase {
         )
     }
 
+    func testDisableAllOverridesSuperfluousDisableCommand() {
+        XCTAssert(
+            !violations(
+                "//swiftlint:disable all\n// swiftlint:disable nesting\nprint(123)\n"
+            ).contains {
+                $0.ruleDescription.identifier == "superfluous_disable_command"
```

Comment:
nitpick: can you check `.isEmpty` instead? This could potentially catch other issues